packages/ai-chat-ui/src/browser/chat-view-language-contribution.ts

```diff
@@ -49,34 +60,77 @@ export class ChatViewLanguageContribution implements FrontendApplicationContribu
     onStart(_app: FrontendApplication): MaybePromise<void> {
         monaco.languages.register({ id: CHAT_VIEW_LANGUAGE_ID, extensions: [CHAT_VIEW_LANGUAGE_EXTENSION] });
 
-        monaco.languages.registerCompletionItemProvider(CHAT_VIEW_LANGUAGE_ID, {
-            triggerCharacters: [PromptText.AGENT_CHAR],
-            provideCompletionItems: (model, position, _context, _token): ProviderResult<monaco.languages.CompletionList> => this.provideAgentCompletions(model, position),
+        // Register all completion providers
+        this.registerCompletionProviders();
+
+        // Register editor commands
+        monaco.editor.registerCommand(VARIABLE_ARGUMENT_PICKER_COMMAND, this.triggerVariableArgumentPicker.bind(this));
+        monaco.editor.registerCommand(VARIABLE_ADD_CONTEXT_COMMAND, (_, ...args) => args.length > 1 ? this.addContextVariable(args[0], args[1]) : undefined);
+    }
+
+    protected registerCompletionProviders(): void {
+        // Register standard completion providers
+        this.registerStandardCompletionProvider({
+            triggerCharacter: PromptText.AGENT_CHAR,
+            getItems: () => this.agentService.getAgents(), // Use function to fetch the latest items
```

Comment:
```suggestion
            getItems: () => this.agentService.getAgents(),
```

Imho this comment doesn't add a lot of information